samples/nrf_desktop/configuration/nrf52_pca20037/app_ZDebug.conf

```diff
@@ -18,6 +18,10 @@ CONFIG_DESKTOP_LED_COLOR_COUNT=1
 
 CONFIG_DESKTOP_POWER_MANAGER_ENABLE=y
 
+CONFIG_DESKTOP_BLE_PEER_CONTROL=y
+CONFIG_DESKTOP_BLE_PEER_CONTROL_BUTTON=0x0211
```

Comment:
Maybe add comment, which key is it?

Reply:
discussed offline. I think this is fine as fn key is not used at all at the moment
when we will start using fn key this part will also be modified